### PR TITLE
Replacing textarea account removal with a link to support.kifi.com

### PR DESCRIPTION
**Pull request by adamjgrant**

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/profile/profile.js [hunk 1]
```diff
@@ -272,35 +272,13 @@ angular.module('kifi')
...
-
-            $analytics.eventTrack('user_clicked_page', {
-              'action': 'clickCloseAccount',
-              'path': $location.path()
-            });
-          }, function () {
-            scope.closeAccountStatus = 'error';
+          $analytics.eventTrack('user_clicked_page', {
+            'action': 'clickCloseAccount',
+            'path': $location.path()
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/profile/profile.js:278]: Leaving this in for now, but I'm going to check with Dafna or Jen to see if they still need this. Might be still good to track.

**Comment by adamjgrant** [comment]
Spoke with Dafne and we're good to review, @andrewconner

**Comment by adamjgrant** [comment]
Oops, forgot to bind the ng-click event...please hold @andrewconner

**Comment by andrewconner** [comment]
Looks good after you have the click event

**Comment by adamjgrant** [comment]
Cool. Mergin.

**Pull request merged**